virt/virtual_machines/vm_networking/virt-accessing-vm-secondary-network-fqdn.adoc

```diff
@@ -0,0 +1,24 @@
+:_content-type: ASSEMBLY
```

Comment:
Just a suggestion. Instead of creating a new assembly, you might want to add these modules to the [existing assembly about secondary networks](https://56582--docspreview.netlify.app/openshift-enterprise/latest/virt/virtual_machines/vm_networking/virt-attaching-vm-multiple-networks.html).

Reply:
@apinnick This content applies to both Linux bridge networks and SR-IOV networks so creating a new assembly made more sense.

Reply:
OK